crates/subspace-proof-of-time/src/pot.rs

```diff
@@ -0,0 +1,101 @@
+//! Common defines.
+
+use sp_core::{Blake2Hasher, Hasher, H256};
+use std::ops::BitXor;
```

Comment:
This library should be `no_std` compatible, all imports should come from `core` and `alloc`.

Reply:
Changed